diff --git a/bun-sql/driver.js b/bun-sql/driver.js
index 3e6456b12641064e3ffbdc4b869c3f9f4cb87590..56474cc6f6198cd9701ac00a2bd3ff84cb14ddcd 100644
--- a/bun-sql/driver.js
+++ b/bun-sql/driver.js
@@ -38,7 +38,7 @@ function construct(client, config = {}) {
 }
 function drizzle(...params) {
   if (typeof params[0] === "string") {
-    const instance = new SQL(params[0]);
+    const instance = new Bun.SQL(params[0]);
     return construct(instance, params[1]);
   }
   if (isConfig(params[0])) {
@@ -47,10 +47,10 @@ function drizzle(...params) {
       return construct(client, drizzleConfig);
     if (typeof connection === "object" && connection.url !== void 0) {
       const { url, ...config } = connection;
-      const instance2 = new SQL({ url, ...config });
+      const instance2 = new Bun.SQL({ url, ...config });
       return construct(instance2, drizzleConfig);
     }
-    const instance = new SQL(connection);
+    const instance = new Bun.SQL(connection);
     return construct(instance, drizzleConfig);
   }
   return construct(params[0], params[1]);
